### ISSUE TITLE
: A11y_AzureDataStudio(Schema,Dacpac,Backup)_ Deploy Dacpac Settings_Screenreader: NVDA is not announcing the label of text box File location.

### ISSUE THREAD

**Issue by mspatil110** [microsoft/azuredatastudio#6748]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
Application Name: Azure Data Studio
Application Version: 1.10.0-insider (user setup)
Commit: 1ff9a2ec4c7b88be01666bdd470bbbb4ab95716e
Date: 2019-08-09T07:49:00.254Z
VS Code 1.37.0
Electron: 4.2.7
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
Tools Name: Accessibility insight tool
MAS References: MAS3.3.2, MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Data-tier application wizard from Context menu.
8. Select first radio button.
9. Navigate till next button & hit enter.
10. Launch NVDA.
11. Navigate up to File Location Text field and observed that NVDA is announcing the label of text box or not.

**Actual:**
NVDA is not announcing the label of text box File location.

**Expected:**
NVDA should announce the label of text box File location.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader users will face difficulty to understand the text box control.
MAS Reference:

**MAS 1.3.1-** https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4} 

**MAS 3.3.2:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/Documents/LCA/MAS%20-%20July%201,%202017%20Release/MAS%203.3.2%20%E2%80%93%20Labels%20or%20Instructions.docx?d=w4b95a70d258c474a820e5c8b79d5c0bf&csf=1&e=sc1Ft4

Attachment Ref:

**Comment by udeeshagautam** [comment]
@kisantia, this should be a simple one so just assigning

**Comment by kisantia** [comment]
Fixed in #6775 when addressing #6731
![image](https://user-images.githubusercontent.com/31145923/63305111-74181c00-c29a-11e9-8692-d0ad45a8534b.png)
